docs/DeveloperGuide.md

```diff
@@ -711,48 +806,49 @@ should end at the destroy marker (X) but due to a limitation of PlantUML, the li
-| `*`      | clumsy tutor                                | undo actions                                  | recover information that I accidentally delete                           |
+| Priority | As a …​                                 | I want to …​                                   | So that I can…​                                                           |                                                 
+| -------- | ------------------------------------------ | ------------------------------------------------- | ---------------------------------------------------------------------------- |
+| `* * *`  | TA using TA<sup>2</sup> for the first time | see all commands available                        | recall commands and use them properly when I forget how to use the app       |
+| `* * *`  | TA                                         | add information of a student                      | -                                                                            |
+| `* * *`  | TA                                         | delete a student                                  | remove student entries that I no longer need                                 |
+| `* * *`  | TA                                         | find a person by name or module                   | locate details of persons without having to go through the entire list       |
+| `* * *`  | TA                                         | assign tasks to students                          | -                                                                            |
+| `* * *`  | TA                                         | delete tasks assigned before                      | remove the outdated assignments for students                                 |
+| `* * *`  | TA                                         | mark students' tasks as done                      | record students' progress more easily                                        |
+| `* * *`  | responsible TA                             | track students' progress on their assignments     | identify and reach out to those who need help                                |
+| `* * *`  | TA                                         | list all students I am teaching                   | ensure I added right and correct number of students                          |
+| `* * *`  | TA                                         | clear all student information in TA<sup>2</sup>   | use TA<sup>2</sup> over many semesters                                       |
+| `* *`    | TA for several modules                     | organize student assignments according to module  | manage assignments of students from different modules in an organised manner |
+| `* *`    | TA with many assignments to manage         | sort student assignments                          | easily see which assignments are not submitted yet                           |
```

Comment:
`sort` is a feature? Maybe the user story can be like below?

I wish to see assignments that need my attention the most at the present moment so that I can prioritise which assignment to attend to.